tests/integration/security/sds_tls_origination/main_test.go

```diff
@@ -20,16 +20,13 @@ import (
 
 	"istio.io/istio/pkg/test/framework"
 	"istio.io/istio/pkg/test/framework/components/istio"
-	"istio.io/istio/pkg/test/framework/label"
 )
 
 var inst istio.Instance
 
 func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
-		Label(label.CustomSetup).
```

Comment:
just curious, the custom label is no longer needed?

Reply:
Yes, I think we can remove it as the [setupConfig()](https://github.com/istio/istio/blob/release-1.7/tests/integration/security/sds_tls_origination/main_test.go#L43) has been removed.